packages/valory/skills/liquidity_provision_abci/__init__.py

```diff
@@ -22,4 +22,4 @@
 from aea.configurations.base import PublicId
 
 
-PUBLIC_ID = PublicId.from_str("valory/liquidity_rebalancing_abci:0.1.0")
+PUBLIC_ID = PublicId.from_str("valory/liquidity_provision_abci:0.1.0")
```

Comment:
How? 😕

Reply:
-> we should add a consistency check in `open-aea` to ensure the public id matches the config! Can you open an issue?